src/io.jl

```diff
@@ -167,26 +168,26 @@ function savetomographyobserver(observer::Observer, output_path::String; model =
       write(fout, "model", "nothing")
     end
     
-    params = results(observer, "parameters")
-    g1 = create_group(fout, "parameters")
-    g1["batchsize"] = params["batchsize"] 
-    g1["nshots"] = params["dataset_size"] 
-    g1["measurement_frequency"] = params["measurement_frequency"]
-    g1["optimizer"] = params["optimizer"][:name]
-    g1["learning_rate"] = params["optimizer"][:η]
-    attributes(g1)["Description"] = "This group contains the training parameters."
-    
     g2 = create_group(fout, "measurements")
-    for (measurement, value) in observer.measurements
-      if measurement != "parameters"
-        g2[measurement] = real.(last(value))
-      end
+    for (measurement, value) in observer.data 
+      # TODO HDF% cannot export complex numbers (?)
+      output = Vector{Float64}(last(value))
+      #if measurement != "parameters"
+      g2[measurement] = output#last(value)
+      #end
     end
     attributes(g2)["Description"] = "This group contains measurements." 
   end
 end
 
-function savecircuitobserver(observer::Observer, output_path::String; model = nothing)
+
+"""
+
+TOMOGRAPHY OBSERVER
+
+"""
+
+function savetomographyobserver(observer::Observer, output_path::String; model = nothing)
```

Comment:
This seems very similar to `savecircuitobserver`, wouldn't it be better to have a single `save(::Observer)` implementation?

Reply:
We could always have an automated feature for skipping measurements that can't be saved by HDF5 (and giving a warning in those cases).

Reply:
Yes, initially they were different because I was also saving parameters for the tomography observer. But with the new Observers it was not very clear to me how to best do that (if doing that at all).